packages/subgraph/src/utils.ts

```diff
@@ -8,6 +8,15 @@ import {
     TokenStatistic,
 } from "../generated/schema";
 
+// TODO when refactoring subgraph to use Bytes id's and immmutable entities
```

Comment:
TODO

Reply:
this will be done in another PR (having issues w/ the Bytes id's though) 🥲